lib/elements/dom-if.html

```diff
@@ -175,12 +175,13 @@
           if (!template) {
             // Wait until childList changes and template should be there by then
             let observer = new MutationObserver(() => {
-              if (this.querySelector('template')) {
-                observer.disconnect();
-                this.__render();
-              } else {
+              // If template tag not found, it will throw
```

Comment:
As an early throw will be done if the first condition has been hit.
I do not think it is necessary to create an else block.

Thanks for your review.